data/environment_variables.yaml

```diff
@@ -315,6 +315,11 @@ variables:
     The base branch that the pull request is targeting.
   modifiable: false
   example: "`\"main\"`, or `\"\"` if not a pull request"
+- name: BUILDKITE_PULL_REQUEST_DRAFT
+  desc: |
+    This environment variable only gets added for a build with a pull request when that pull request is a draft.
```

Comment:
How about? 

```suggestion
    Set to `true` when the pull request is a draft. 
```

Reply:
I would add that the variable will be missing otherwise

Reply:
Why don't we set it to false in that scenario? :-p But yeah, makes sense.

Reply:
It's on purpose :P the var will only exists when is true because there is a size limit that we may hit so we've being careful with new en vars

Reply:
TIL. Thanks @pzeballos !

Reply:
Updated. Plz take a look & let's merge.